coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -598,7 +609,21 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
   private get directorFirstNameRules (): Array<Function> {
     return [
       v => !!v || 'A first name is required',
-      v => !/\s/g.test(v) || 'Invalid spaces'
+      v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+      v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+      v => !/\s\s/g.test(v) || 'Invalid word spacing' // multiple inline spaces
```

Comment:
Did we want to allow multiple names in the single name input fields?

Reply:
Yes. "Ann Marie" (or Bobbi Jo, or take your pick) are valid first names. This rule allows them but only if separated by 1 whitespace character.

Reply:
Makes sense!👍